### ISSUE TITLE
Compliance with GDPR

### ISSUE THREAD

**Issue by ssj7** [lilili87222/admob-for-flash#98]
Hi, is the ane complaint with GDPR? I couldn't find any info on GDPR.

Thanks a lot for the hard work. Awaiting your reply.

**Comment by Gusher** [comment]
Hi lilili87222 , is that code already in the latest release or we need to add? Would like more info on how to implement, especially, how to add less code in a frame without using an AS file... 
=====
import so.cuo.platform.nativeextensions.*; 
import so.cuo.platform.nativeextensions.events.*;
if(AdMob.isSupported)
{
AdMob.init("you must type your ad unit id here"
extraParam=new ExtraParameter();
extraParam.nonPersonalizedAds=true;//if want to load non Personalized ads set true GDPR setting
Admob.getInstance().showBanner(AdmobSize.BANNER_320x50,AdmobPosition.BOTTOM_CENTER,80,extraParam);
);
// remove this next when you're done testing!
AdMob.enableTestDeviceIDs(AdMob.getCurrentTestDeviceIDs());
AdMob.showAd(AdMobAdType.SMART_BANNER,AdMobAlignment.CENTER,AdMobAlignment.TOP);
}
====

**Comment by lilili87222** [comment]
Yes,already in the latest release

**Comment by Gusher** [comment]
Ok, thanks.... do you have a payed premium version of this ANE?

**Comment by lilili87222** [comment]
Maybe make a premium version contain more function

**Comment by Gusher** [comment]
Yes, would be nice...with better step by step documentation.

**Comment by lilili87222** [comment]
```
extraParam=new ExtraParameter();
extraParam.nonPersonalizedAds=true;//if want to load non Personalized ads set true GDPR setting

	ad.cacheInterstitial(extraParam);
```

**Comment by hadisn** [comment]
Tried but no luck, i will try again and let you know if works. Thank you

**Comment by hadisn** [comment]
Yes, it works. I needed to reset my advertising id because too many testing ads. Thank you very much :)

**Comment by hadisn** [comment]
Apk have 5mb more after compiling, is there option to compress it, or remove if there something not needed?

**Comment by ssj7** [comment]
Thanks
